spec/lib/reports/careplus_exporter_spec.rb

```diff
@@ -10,25 +10,50 @@
     )
   end
 
+  let(:programme) { create(:programme, programme_type) }
+  let(:programme_type) { nil }
+  let(:parsed_csv) { CSV.parse(csv) }
+  let(:programmes) { [programme] }
+  let(:session) { create(:session, organisation:, programmes:, location:) }
+  let(:location) do
+    create(
+      :school,
+      gias_local_authority_code: 123,
+      gias_establishment_number: 456
+    )
+  end
+  let(:headers) { parsed_csv.first }
+  let(:organisation) do
+    create(:organisation, careplus_venue_code: "ABC", programmes:)
+  end
+
+  def create_vaccination_record(options = {})
+    patient_session =
+      create(
+        :patient_session,
+        :consent_given_triage_not_needed,
+        programmes:,
+        session:
+      )
+
+    vaccine = create(:vaccine, programme:)
```

Comment:
I think these can be pulled out in to their own `let` as they don't rely on the `options` passed in.